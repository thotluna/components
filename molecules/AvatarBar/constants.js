export const options = {
  sizes: ['sm', 'lg'],
}
