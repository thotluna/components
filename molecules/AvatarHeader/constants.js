export const options = {
  size: ['sm', 'md'],
}
