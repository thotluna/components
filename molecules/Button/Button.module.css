.button {
  display: flex;
  width: 100%;
  min-width: max-content;
  align-items: center;
  justify-content: center;
  padding: 12px 17px;
  border-radius: var(--border-radius-full);
  font-family: var(--font-family-title);
  font-size: 16px;
  outline: none;
}

.button:focus,
.button:hover {
  box-shadow: var(--box-shadow-in), var(--box-shadow-out);
}

.button:active {
  box-shadow: var(--box-shadow-dark-in), var(--box-shadow-dark-out);
}

.color-base {
  border: 2px solid var(--button-base-color-border);
  background-color: var(--button-base-background);
  color: var(--button-base-color);
}

.color-primary {
  border: 2px solid var(--button-primary-color-border);
  background-color: var(--button-primary-background);
  color: var(--button-primary-color);
}

.color-secondary {
  border: 2px solid var(--button-primary-color-border);
  background-color: var(--button-secondary-background);
  color: var(--button-primary-color);
}

.color-error {
  border: 2px solid var(--button-primary-color-border);
  background-color: var(--button-secondary-background);
  color: var(--color-font-error);
}

.is-compact {
  max-width: max-content;
}

.button.is-round {
  width: 36px !important;
  height: 36px !important;
  padding: 0;
  border: var(--border-width-thick) solid var(--color-base-transparent);
}
