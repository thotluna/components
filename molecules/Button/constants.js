export const options = {
  colors: ['base', 'primary', 'secondary'],
}
