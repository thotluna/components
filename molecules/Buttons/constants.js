export const options = {
  size: ['full', 'short', 'mini'],
  color: ['primary', 'secondary', 'inverted'],
}
