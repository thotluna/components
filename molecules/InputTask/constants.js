export const status = {
  STATUS_STANDBY: 'standby',
  STATUS_CREATING: 'creating',
}

export const options = {
  status: [status.STATUS_STANDBY, status.STATUS_CREATING],
}
