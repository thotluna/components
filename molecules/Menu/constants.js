export const options = {
  position: ['left', 'right'],
}
