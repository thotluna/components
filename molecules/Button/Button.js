import React from 'react'
import PropTypes from 'prop-types'
import styles from './Button.module.css'
import { options } from './constants'
import { getClasses } from '../../helpers/styles'
import Icon, { options as optionIcon } from '../../atoms/Icon'
import Spacer from '../../layout/Spacer'

export const Button = ({
  value,
  id,
  ariaLabel,
  color,
  icon,
  iconRight,
  isCompact,
  handelOnClick,
}) => {
  const getStyles = getClasses(styles)({ color })
  return (
    <button
      id={id}
      aria-label={ariaLabel}
      className={getStyles('button', ['color'], {
        'is-compact': isCompact,
        'is-round': !value && icon,
      })}
      onClick={handelOnClick}
    >
      {icon && (
        <Icon
          type={icon}
          color={
            color === 'base' ? 'base' : color === 'error' ? 'error' : 'primary'
          }
          size="md"
        />
      )}
      {icon && value && <Spacer.Vertical size="sm" />}
      {value}
      {iconRight && value && (
        <>
          <Spacer.Vertical size="sm" />
          <Icon
            type={iconRight}
            color={
              color === 'base'
                ? 'base'
                : color === 'error'
                ? 'error'
                : 'primary'
            }
          />
        </>
      )}
    </button>
  )
}

Button.propTypes = {
  id: PropTypes.string.isRequired,
  ariaLabel: PropTypes.string.isRequired,
  handelOnClick: PropTypes.func,
  value: PropTypes.string,
  icon: PropTypes.oneOf(optionIcon.types),
  iconRight: PropTypes.oneOf(optionIcon.types),
  color: PropTypes.oneOf(options.colors),
  isCompact: PropTypes.bool,
}

Button.defaultProps = {
  color: 'base',
  isCompact: false,
}

export default Button
