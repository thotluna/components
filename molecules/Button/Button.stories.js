import React from 'react'
import Button from '.'
import { options } from './constants'
import { options as iconOptions } from '../../atoms/Icon'
import { getVariusTemplate, getOptionsArgTypes } from '../../helpers/storybook'

const Template = (args) => <Button {...args} />
const ListTemplate = getVariusTemplate(Button)

export default {
  title: 'Molecules/Button',
  button: Button,
  args: {
    id: '#1',
    ariaLabel: 'Butron Text',
    value: 'BUTTON',
  },
  argTypes: {
    value: { control: 'text' },
    isCompact: { control: 'boolean' },
    icon: getOptionsArgTypes(iconOptions.types),
    iconRight: getOptionsArgTypes(iconOptions.types),
    color: getOptionsArgTypes(options.colors),
    onClick: { action: 'clicked' },
  },
}

export const Default = Template.bind({})

export const Colors = ListTemplate.bind({})
Colors.args = {
  items: options.colors.map((color, index) => ({ color, id: `${index}` })),
}

export const ColorsCompact = ListTemplate.bind({})
ColorsCompact.args = {
  items: options.colors.map((color, index) => ({
    color,
    id: `${index}`,
    isCompact: true,
  })),
}

export const IconOnly = ListTemplate.bind({})
IconOnly.args = {
  items: iconOptions.types
    .map((icon, index) => {
      return options.colors.map((color, i) => {
        return {
          id: `${index}-${i}`,
          icon,
          color,
          value: '',
        }
      })
    })
    .flat(),
}
IconOnly.parameters = {
  __sb: { fd: 'row' },
}

export const ColorsIcon = ListTemplate.bind({})
ColorsIcon.args = {
  items: options.colors.map((color, i) => {
    return {
      id: `ColorAndIcon-${i}`,
      icon: 'check',
      color,
    }
  }),
}
