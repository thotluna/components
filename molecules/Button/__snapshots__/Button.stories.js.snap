// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Molecules/Button Colors 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "background": "",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
      "padding": "2em",
    }
  }
>
  <button
    aria-label="Butron Text"
    className="button color-base"
    id="0"
  >
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary"
    id="1"
  >
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary"
    id="2"
  >
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error"
    id="3"
  >
    BUTTON
  </button>
</div>
`;

exports[`Storyshots Molecules/Button Colors Compact 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "background": "",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
      "padding": "2em",
    }
  }
>
  <button
    aria-label="Butron Text"
    className="button color-base is-compact"
    id="0"
  >
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-compact"
    id="1"
  >
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-compact"
    id="2"
  >
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-compact"
    id="3"
  >
    BUTTON
  </button>
</div>
`;

exports[`Storyshots Molecules/Button Colors Icon 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "background": "",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
      "padding": "2em",
    }
  }
>
  <button
    aria-label="Butron Text"
    className="button color-base"
    id="ColorAndIcon-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    <div
      className="spacer vertical"
      style={
        Object {
          "height": "100%",
          "maxHeight": "auto",
          "width": 15,
        }
      }
    />
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary"
    id="ColorAndIcon-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    <div
      className="spacer vertical"
      style={
        Object {
          "height": "100%",
          "maxHeight": "auto",
          "width": 15,
        }
      }
    />
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary"
    id="ColorAndIcon-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    <div
      className="spacer vertical"
      style={
        Object {
          "height": "100%",
          "maxHeight": "auto",
          "width": 15,
        }
      }
    />
    BUTTON
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error"
    id="ColorAndIcon-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    <div
      className="spacer vertical"
      style={
        Object {
          "height": "100%",
          "maxHeight": "auto",
          "width": 15,
        }
      }
    />
    BUTTON
  </button>
</div>
`;

exports[`Storyshots Molecules/Button Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "background": "",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
      "padding": "2em",
    }
  }
>
  <button
    aria-label="Butron Text"
    className="button color-base"
    id="#1"
  >
    BUTTON
  </button>
</div>
`;

exports[`Storyshots Molecules/Button Icon Only 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "background": "",
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
      "padding": "2em",
    }
  }
>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="0-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(1 1)"
        >
          <path
            d="M9.5.5l9 16H.5zM9.5 10.5v-5"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
          <circle
            cx={9.5}
            cy={13.5}
            fill="currentColor"
            r={1}
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="0-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(1 1)"
        >
          <path
            d="M9.5.5l9 16H.5zM9.5 10.5v-5"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
          <circle
            cx={9.5}
            cy={13.5}
            fill="currentColor"
            r={1}
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="0-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(1 1)"
        >
          <path
            d="M9.5.5l9 16H.5zM9.5 10.5v-5"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
          <circle
            cx={9.5}
            cy={13.5}
            fill="currentColor"
            r={1}
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="0-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(1 1)"
        >
          <path
            d="M9.5.5l9 16H.5zM9.5 10.5v-5"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
          <circle
            cx={9.5}
            cy={13.5}
            fill="currentColor"
            r={1}
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="1-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="1-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="1-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="1-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.5 11.5l3 3 8.028-8"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="2-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M7.5 7.5l6 6M13.5 7.5l-6 6"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="2-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M7.5 7.5l6 6M13.5 7.5l-6 6"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="2-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M7.5 7.5l6 6M13.5 7.5l-6 6"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="2-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M7.5 7.5l6 6M13.5 7.5l-6 6"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="3-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height="21"
        stroke="currentColor"
        viewBox="0 0 21 21"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="m8.5.5-4 4-4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
          transform="translate(6 8)"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="3-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height="21"
        stroke="currentColor"
        viewBox="0 0 21 21"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="m8.5.5-4 4-4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
          transform="translate(6 8)"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="3-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height="21"
        stroke="currentColor"
        viewBox="0 0 21 21"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="m8.5.5-4 4-4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
          transform="translate(6 8)"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="3-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height="21"
        stroke="currentColor"
        viewBox="0 0 21 21"
        width="21"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="m8.5.5-4 4-4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
          transform="translate(6 8)"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="4-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 14.5l-4-4 4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="4-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 14.5l-4-4 4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="4-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 14.5l-4-4 4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="4-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 14.5l-4-4 4-4"
          fill="none"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="5-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        fill="currentColor"
        height={21}
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(2 2)"
        >
          <path
            d="M8.5 9.5c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm-4 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm8 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1z"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="5-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        fill="currentColor"
        height={21}
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(2 2)"
        >
          <path
            d="M8.5 9.5c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm-4 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm8 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1z"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="5-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        fill="currentColor"
        height={21}
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(2 2)"
        >
          <path
            d="M8.5 9.5c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm-4 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm8 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1z"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="5-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        fill="currentColor"
        height={21}
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          transform="translate(2 2)"
        >
          <path
            d="M8.5 9.5c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm-4 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1zm8 0c.5 0 1-.5 1-1s-.5-1-1-1-.999.5-.999 1 .499 1 .999 1z"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-base is-round"
    id="6-0"
  >
    <span
      className="icon color-base size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M6.5 10.5l4 4.232 4-4.191M10.5 3.5v11M4.5 17.5h12"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-primary is-round"
    id="6-1"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M6.5 10.5l4 4.232 4-4.191M10.5 3.5v11M4.5 17.5h12"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-secondary is-round"
    id="6-2"
  >
    <span
      className="icon color-primary size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M6.5 10.5l4 4.232 4-4.191M10.5 3.5v11M4.5 17.5h12"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
  <button
    aria-label="Butron Text"
    className="button color-error is-round"
    id="6-3"
  >
    <span
      className="icon color-error size-sm"
    >
      <svg
        height={21}
        stroke="currentColor"
        viewBox="0 0 21 21"
        width={21}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          strokeLinecap="round"
          strokeLinejoin="round"
        >
          <path
            d="M6.5 10.5l4 4.232 4-4.191M10.5 3.5v11M4.5 17.5h12"
          />
        </g>
      </svg>
    </span>
    
    
  </button>
</div>
`;
